docs/src/_guide/integration.md

```diff
@@ -15,69 +15,17 @@ and keeps track of [the incremental replication state](#incremental-replication-
 so that subsequent pipeline runs with the same state ID will always pick up right where
 the previous run left off.
 
-You can run EL(T) pipelines using [`meltano elt`](/reference/command-line-interface#elt).
+You can run EL(T) pipelines using [`meltano run`](/reference/command-line-interface#elt).
 If you encounter some trouble running a pipeline, read our [troubleshooting tips](#troubleshooting) for some errors commonly seen.
 
 ## Plugin configuration
 
-As described in the [Configuration guide](/guide/configuration#configuration-layers), [`meltano elt`](/reference/command-line-interface#elt) will determine the configuration of the extractor, loader, and (optionally) transformer by looking in [**the environment**](/guide/configuration#configuring-settings), your project's [**`.env` file**](/concepts/project#env), the [system database](/concepts/project#system-database), and finally your [**`meltano.yml` project file**](/concepts/project#meltano-yml-project-file), falling back to a default value if nothing was found.
+As described in the [Configuration guide](/guide/configuration#configuration-layers), [`meltano run`](/reference/command-line-interface#run) will determine the configuration of the extractor, loader, and (optionally) transformer by looking in [**the environment**](/guide/configuration#configuring-settings), your project's [**`.env` file**](/concepts/project#env), the [system database](/concepts/project#system-database), and finally your [**`meltano.yml` project file**](/concepts/project#meltano-yml-project-file), falling back to a default value if nothing was found.
```

Comment:
The configuration layers section of the configuration guide describes these things for `meltano elt`, not `meltano run`. Even if it were updated to say `meltano run` instead of `meltano elt`, I'm not sure that that would make it true.

Reply:
@sbalnojan Two possible quick fixes:
- Update `guide/configuration#configuration-layers` to use `meltano run` instead of `meltano elt` (preferable).
- Remove the link to `guide/configuration#configuration-layers`. The rest of the paragraph seems mostly fine (if a bit light on details) without it.

In either case you'll need to rephrase this part:
> will determine the configuration of the extractor, loader, and (optionally) transformer